src/ric/d_18C40.c

```diff
@@ -0,0 +1,330 @@
+    0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000,
+    0x00000000, 0x00000000, 0x00000000, 0x00000000,
+};
+
+s16 D_80154FBC[][10] = {
+    0x0000, 0x0000, 0x0000, 0x015F, 0x0100, 0x0100, 0x0100, 0x0060, 0x0031,
+    0x0200, 0x0000, 0x0002, 0x0006, 0x015F, 0x0100, 0x0100, 0x0100, 0x0040,
+    0x0031, 0x0200, 0x0000, 0x0000, 0x0000, 0x015F, 0x0100, 0x7000, 0x7000,
+    0x0040, 0x0031, 0x0200, 0x0000, 0x0000, 0x0000, 0x015F, 0x0100, 0x7000,
+    0x0200, 0x0010, 0x0031, 0x0200, 0x0000, 0x0000, 0x0000, 0x015F, 0x0100,
+    0x7000, 0x7000, 0x0010, 0x0031, 0x0200, 0x0000, 0x0000, 0x0000, 0x015F,
+    0x0100, 0x7000, 0x0200, 0x7000, 0x0031, 0x0200, 0x0000, 0x0000, 0x0000,
+    0x015F, 0x0100, 0x7000, 0x0100, 0x7002, 0x0031, 0x0100, 0x0000, 0x0000,
+    0x0000, 0x015F, 0x0100, 0x0100, 0x7000, 0x0070, 0x0031, 0x0200, 0x0000,
+    0x0000, 0x0000, 0x015F, 0x0100, 0x7000, 0x0200, 0x7006, 0x0031, 0x0200,
+    0x0000, 0x0000, 0x0000, 0x015F, 0x0200, 0x0200, 0x0400, 0x7001, 0x0031,
+    0x0200, 0x0000, 0x0000, 0x0000, 0x0166, 0x0100, 0x0100, 0x0100, 0x7005,
+    0x0031, 0x0200, 0x0000, 0x0000, 0x0000, 0x0102, 0x0100, 0x7000, 0x7000,
+    0x0040, 0x0031, 0x0200, 0x0000, 0x0000, 0x0000, 0x015F, 0x0100, 0x7100,
+    0x0100, 0x7005, 0x0031, 0x0200, 0x0000, 0x0000, 0x0000, 0x015F, 0x0100,
+    0x0100, 0x0100, 0x7005, 0x0031, 0x0200, 0x0000, 0x0000, 0x0000, 0x0102,
+    0x0100, 0x7000, 0x0200, 0x0040, 0x0031, 0x0200, 0x0000, 0x0000, 0x0000,
+    0x0102, 0x0100, 0x7000, 0x0100, 0x0040, 0x0031, 0x0200, 0x0000, 0x0000,
+    0x0000, 0x0102, 0x0100, 0x0100, 0x7000, 0x0000, 0x0031, 0x0200, 0x0000,
+    0x0000, 0x0000, 0x0102, 0x0400, 0x0100, 0x0400, 0x0000, 0x0031, 0x0200,
+    0x0000, 0x0000, 0x0000, 0x015F, 0x0100, 0x0100, 0x7000, 0x0000, 0x0031,
+    0x0200, 0x0000, 0x0000, 0x0000, 0x015F, 0x0200, 0x0800, 0x0800, 0x7007,
+    0x0031, 0x0200, 0x0000, 0x0000, 0x0000, 0x0102, 0x0100, 0x0100, 0x0100,
+    0x0000, 0x0031, 0x0200, 0x0000, 0x0000, 0x0000, 0x015F, 0x00C0, 0x00C0,
+    0x0100, 0x0010, 0x0031, 0x0200, 0x0008, 0x0000, 0x0008, 0x0165, 0x0100,
+    0x0100, 0x0100, 0x7007, 0x0051, 0x0200, 0x0000, 0x0000, 0x0000, 0x0168,
+    0x7100, 0x7100, 0x0200, 0x7007, 0x0031, 0x0200,
+};
+
+Point16 D_8015519C[] = {
+    {0, 0},         {256, 256},   {256, 256},   {1280, 2560}, {256, 256},
+    {256, 0},       {0, 0},       {256, 28672}, {28672, 512}, {0, 0},
+    {28928, 28928}, {256, 4096},  {0, 0},       {28928, 256}, {28672, 2048},
+    {0, 0},         {256, 28928}, {256, 1024},  {1280, 2560}, {256, 256},
+    {256, 256},     {1280, 2560}, {256, 512},   {28928, 256}, {-2, -24},
+    {0, -8},        {2, 8},       {0, 24},      {6, -12},     {8, 0},
+    {7, 12},        {-8, -12},    {-9, 0},      {-7, 12},     {-14, -6},
+    {-15, 7},       {13, -7},     {16, 8},      {-7, -22},    {6, -21},
+    {-8, 21},       {7, 22}};
+
+Point16* D_80155244[] = {
+    &D_8015519C[0x18], // + 0x60
+    &D_8015519C[0x1C], // + 0x70
```

Comment:
```suggestion
    &D_8015519C[0x1C],
```
any reason to leave those comments? They seem to be redundant with the index value already defined

Reply:
As you can see, and Sozud pointed out, there's something wrong with that array configuration so i'm leaving those comments there to help resolution.